Automattic/wp-calypso
# ExternalLink to packages (Step 3): Update references to use new package component

Related to https://github.com/Automattic/wp-calypso/issues/88181
Branched from https://github.com/Automattic/wp-calypso/pull/88290

We should wait for https://github.com/Automattic/wp-calypso/pull/88290 to sit in production for several days before merging this PR. It will allow us to see if there were any problems with our migration.

## Proposed Changes

- Repoints any references to `ExternalLink` and `ExternalLinkWithTracking` from Calypso components to `@automattic/components`
- Fixes some missing `localizeUrl` calls flagged during the commit
- This is a change that touches many files. We effectively made this change in the background with https://github.com/Automattic/wp-calypso/pull/88290, so all `ExternalLink` and `ExternalLinkWithTracking` components are running the new/relocated code already. This PR repoints the reference from the wrapper to the "real" code, so I don't expect this to cause any issues.

## Testing Instructions



- Make sure tests pass.
- Do a spot-check of `ExternalLink` instances to ensure they still look/work as expected. See https://github.com/Automattic/wp-calypso/pull/88290 for some examples.

## Pre-merge Checklist
